### PR TITLE
Top albums

### PR DESCRIPTION
**Pull request by taymor10**
@mjfaade10 how do you feel about these top 6 albums?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,11 @@ Friday
...
 Enemy
 Nocturnal Animals
+
+Top five albums:
+Frank Ocean - Blonde
+Drake - More Life
+Travis Scott - Astroworld
+Wiz Khalifa - Kush & Orange Juice
+Mac Miller - KIDS
+Partynextdoor - Partynextdoor
```
Comment (mjfaade10) [inline README.md]: Partynextdoor???? He's trash. Change this lol

File: README.md [hunk 2]
```diff
@@ -15,3 +15,11 @@ Friday
 The Conjuring
 Enemy
 Nocturnal Animals
+
+Top five albums:
+Frank Ocean - Blonde
+Drake - More Life
+Travis Scott - Astroworld
+Wiz Khalifa - Kush & Orange Juice
```
Comment (mjfaade10) [inline README.md:23]: Wiz mixtape is a classic for sure

**Review by mjfaade10 — CHANGES_REQUESTED** [review]
@mjfaade10 Update Partynextdoor and we should be good to go!

File: README.md (hunk 1 above)
Comment (taymor10) [reply README.md]: @mjfaade10 this has been updated!

File: README.md [hunk 3]
```diff
@@ -22,4 +22,4 @@ Drake - More Life
 Travis Scott - Astroworld
 Wiz Khalifa - Kush & Orange Juice
 Mac Miller - KIDS
-Partynextdoor - Partynextdoor
+Missy Elliott - Supa Dupa Fly
```
Comment (mjfaade10) [inline README.md:25]: Love it!

**Review by mjfaade10 — APPROVED** [review]
Looks good!

**Comment by mjfaade10** [comment]
general comment

**Review by mjfaade10 — CHANGES_REQUESTED** [review]
can we do top 10 instead of 5?

**Pull request merged**